GOVERNANCE.md

```diff
@@ -122,9 +122,21 @@ These members will be elected for a term of 1 year as follows:
 
 ## Approving and Onboarding Regular Members
 
-Regular members can self-nominate by opening a PR to add themselves to the Regular member list in the [README.md][README]. The PR should include an indication of which project they have been active in for 3 months as per the [CPC charter][cpc charter regular members].
+While our goal in the OpenJS Foundation is to do as much or our work in the public, on occasion, there are private matters and private materials. Voting and Regular members are included in these private matters and have access to private materials. Therefore, we have some requirements in place to ensure a level of involvement in our work and to hold an expectation of professionalism.
```

Comment:
```suggestion
Our goal in the OpenJS Foundation is to do as much or our work in the public, however, on occasion there are matters and materials that must be kept private. This private information is shared with Voting and Regular members. As a result we have requirements in place in order to ensure that Voting and Regular members are known and active in the the projects they represent or know to the CPC and active it its work. It is important to note however, that Observers can participate in almost all aspects of the work of the CPC except those related to private information and aspects involving private information are limited.
```

Reply:
```suggestion
Our goal in the OpenJS Foundation is to do most of our work in public. On occasion, there are matters and materials that must be kept private and shared only with Voting and Regular members. As a result we have requirements in place in order to ensure that Voting and Regular members are known and active in the the projects they represent or known to the CPC and active in its work. Observers can participate in almost all aspects of the work of the CPC except those infrequent matters related to private information.
```

Reply:
That was riffing off of @mhdawson, tightening the wording, and fixing a few typos.

Reply:
I'll take this one. :)